packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -1386,7 +1401,11 @@ export class AnalyzerService {
 
                 // If this is taking a long time, log an error to help the user
                 // diagnose and mitigate the problem.
-                if (secondsSinceStart >= longOperationLimitInSec && nFilesVisited >= nFilesToSuggestSubfolder) {
```

Comment:
Since I'm changing the pyright-internal could you please verify if this the way to go? In such a how do you keep parity with any upstream changes in Pyright? Cheers!

Reply:
editing files in `pyright-internal` directly is fine, i just merge upstream changes and resolve conflicts like normal